pennylane/beta/devices/default_tensor.py

```diff
@@ -164,6 +164,18 @@ def __init__(self, wires, shots=1000, representation="exact", contraction_method
         self._rep = representation
         self._contraction_method = contraction_method
         self.reset()
+        self.analytic = False
```

Comment:
Is it ok if I add this attribute? It does not change anything, but allows to run some more device tests. The final decision of how the ``analytic`` attribute will be set/renamed/used is still outstanding, but an overall strategy is in our near-term goals.

Reply:
:+1:

Reply:
:rofl: